org.knime.knip.base/src/org/knime/knip/base/nodes/proc/resizer/ResizerNodeFactory.java

```diff
@@ -249,7 +249,6 @@ protected String getDefaultSuffixForAppend() {
 
             private final SettingsModelDouble m_inputFactorModel = createAffectAllDimensionModel();
 
-            //ADDED BY ME
```

Comment:
That was just an annotation in the code to see something I added, to be able to find it again easier. I forgot to delete it before the commit, so i deleted it